### PR TITLE
Exomeseq data demo

### PR DESCRIPTION
**Pull request by jaybee84**
New PR after incorporating the changes suggested in the earlier PR. Output lives here: syn20370599

requesting review from @sgosline and @allaway

### CODE REVIEW

**Comment by allaway** [comment]
is there a difference between this and #12 ? They seem like they are the same changes to me. Just checking before diving in.

**Comment by jaybee84** [comment]
#12  and #13 are two separate datasets... one is whole genome and the other is exome seq... both have different outputs (albeit parallel structures). 

I have kept these datasets separate since intronic variants are missing from exomeseq data.

**Comment by allaway** [comment]
I see, i just got confused because "R_demos/1-rnaseq-data-introduction.Rmd" has the same commits in both PRs. Thanks for clarifying!

**Comment by jaybee84** [comment]
Apologies for the confusion! Yeah, the rna-seq-intro file showed up in both due to parallel PRs ...

File: R_demos/1-rnaseq-data-introduction.Rmd [hunk 1]
```diff
@@ -48,7 +48,7 @@ library(tidyverse)
 
 ## Logging in to Synapse
 
-Please download your favorite Synapse client: [R](https://r-docs.synapse.org/) or [Python client](https://python-docs.synapse.org//build/html/index.html). Refer to the necessary docs [synapser docs](https://r-docs.synapse.org/articles/synapser.html#connecting-to-synapse) or [Python](https://python-docs.synapse.org//build/html/index.html) to successfully log in to synapse and start using its various analytical tools.
+Please download your favorite Synapse client: [R](https://r-docs.synapse.org/) and refer to the necessary docs [synapser docs](https://r-docs.synapse.org/articles/synapser.html#connecting-to-synapse) to successfully log in to synapse and start using its various analytical tools.
```
Comment (sgosline) [inline R_demos/1-rnaseq-data-introduction.Rmd]:
```suggestion
Please download the Synapse R client: [R](https://r-docs.synapse.org/) and refer to the necessary docs [synapser docs](https://r-docs.synapse.org/articles/synapser.html#connecting-to-synapse) to successfully log in to synapse and start using its various analytical tools.
```

File: R_demos/4-exomeseq-introduction.Rmd [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+    df_print: paged 
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Getting Started
+
+This document aims to guide the Research Case participants, who have limited exposure to Synapse as a platform and clinical/biological data, through exploration and analysis of the available exomeSeq datasets. The available dataset has been carefully curated to include genomic variant (exomeSeq) data from normal and three different tumor types from two different study initiatives into a single platform (a harmonized dataset).
```
Comment (sgosline) [inline R_demos/4-exomeseq-introduction.Rmd]: we might need to update `Research Case` to `Hackathon` at some point. @allaway have we come to a conclusion about this?

File: R_demos/4-exomeseq-introduction.Rmd [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+
+#Plot Metadata
+exome_data %>% group_by(Tumor_Sample_Barcode) %>%
+ggplot(aes(x=sex))+
+    geom_bar(aes(x=sex,fill=as.factor(tumorType)),position='dodge')+
+    scale_fill_brewer(palette="Spectral")+
+    ggtitle('Exome-seq sample distribution')
+```
+
+We can also look at the different types of mutations observed for various genes according to the tumor type.
```
Comment (sgosline) [inline R_demos/4-exomeseq-introduction.Rmd]: Can you provide a link to a description of mutation types defined by VEP?

**Review by sgosline — CHANGES_REQUESTED** [review]
Just a few things. looking good.

File: R_demos/4-exomeseq-introduction.Rmd (hunk 2 above)
Comment (jaybee84) [reply R_demos/4-exomeseq-introduction.Rmd]: I already pushed a change to reflect hackathon after seeing the messages on SVAI slack... wonder why it is not reflected here ... :(

File: R_demos/4-exomeseq-introduction.Rmd (hunk 3 above)
Comment (jaybee84) [reply R_demos/4-exomeseq-introduction.Rmd]: Added the necessary info

**Review by jaybee84 — COMMENTED** [review]
added info regarding mutation types recognized by VEP

**Review by sgosline — APPROVED** [review]
looks great!

File: R_demos/4-exomeseq-introduction.Rmd (hunk 2 above)
Comment (allaway) [reply R_demos/4-exomeseq-introduction.Rmd]: Yep, hackathon is the term. looks like this was already changed :)

**Pull request merged**